src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -199,7 +199,8 @@
     //  ]
     //},
     //"OrchardCore_HealthChecks": {
-    //  "Url": "/health/live"
+    //  "Url": "/health/live",
+    // "ShowDetails": true
```

Comment:
Document this under https://docs.orchardcore.net/en/latest/docs/reference/modules/HealthChecks/.